content/blog/2019/10/2019-10-04-vscode-caseStudy.adoc

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Needs file name updated for blog date, probably need to cherry-pick this commit in as well https://github.com/jenkins-infra/jenkins.io/pull/2734/commits/8b148229549a07e4717962f2ddac27c9d01c8328

Reply:
Is that commit already on master?

Reply:
yes it is now, please update the file name to tomorrows date